### PR TITLE
Add nllb translator to the comparison

### PR DESCRIPTION
**Pull request by marco-c**
Part of https://github.com/mozilla/firefox-translations-training/issues/179

### CODE REVIEW

**Comment by marco-c** [comment]
@eu9ene FYI, I found a "hack" to avoid the 5 hours deadline. I slowly add languages to the list of supported languages by the translator, adding a few in each commit.

**Comment by eu9ene** [comment]
> @eu9ene FYI, I found a "hack" to avoid the 5 hours deadline. I slowly add languages to the list of supported languages by the translator, adding a few in each commit.

This is an acceptable workaround for now. NLLB supports a lot of languages though, so it's better to leave this list empty in the end (meaning translate all)  so that when we add new languages we don't need to update it.

**Review by eu9ene — DISMISSED** [review]
Please also document the new translators as a part of these pull requests. See https://github.com/mozilla/firefox-translations-models/issues/102

**Comment by eu9ene** [comment]
I also noticed that we don't run "Comet compare" for the new translators. It is [recommended](https://github.com/Unbabel/COMET/tree/master#comparing-multiple-systems) to run it by the authors. Comet results are very weird for Argos and conflict with BLEU ones, so maybe running it might clarify things.

**Comment by marco-c** [comment]
"comet compare" is useful when the scores are pretty close, but it won't help in general.
We'll have to investigate closer the weirdness in some of the Comet scores.

**Comment by marco-c** [comment]
> Please also document the new translators as a part of these pull requests. See https://github.com/mozilla/firefox-translations-models/issues/102

Where were you thinking they should be documented?

**Comment by eu9ene** [comment]
> > Please also document the new translators as a part of these pull requests. See #102
> 
> Where were you thinking they should be documented?

The translators are already documented in the report templates:
https://github.com/mozilla/firefox-translations-models/blob/main/evals/eval/bleu-results.md#translators
https://github.com/mozilla/firefox-translations-models/blob/main/evals/eval/comet-results.md

And slightly in the readme: https://github.com/mozilla/firefox-translations-models/tree/main#automatic-quality-evaluation

So we should update them with the new ones.

File: evals/translators/nllb.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+import torch
+from transformers import AutoModelForSeq2SeqLM, AutoTokenizer
+import sys
+
+from tqdm import tqdm
+import toolz
+
+
+lang_to_lang_code = {
```
Comment (eu9ene) [inline evals/translators/nllb.py:11]: Would it be possible to extend this list to all the languages supported by NLLB? I'm sure we'll keep forgetting to update it when adding new languages.

Comment (marco-c) [reply evals/translators/nllb.py:11]: I wasn't able to find a trusted comprehensive mapping for this. There seems to be a list from language to nllb language code in https://github.com/winstxnhdw/nllb-api, but we'll need to manually map.
I will file a follow-up to fill this map up.

Comment (eu9ene) [reply evals/translators/nllb.py:11]: ok, you could add a TODO for this in the code then

**Review by eu9ene — APPROVED** [review]
Looks good, but let's make sure we address the issue with language mapping for NLLB in the future

**Pull request merged**